modules/ROOT/pages/install-rtfctl.adoc

```diff
@@ -556,6 +556,25 @@ Replicas:          1 desired | 1 updated | 1 total | 0 available | 1 unavailable
   hello-world-77f5897949-z54gb   Pending
 ----
 
+== Auditing rtfctl Operations
+
+The `rtfctl` command line tool (from vX.X.X and above) audits the `rtfctl` operations on the cluster and it helps cluster administrators and you to track operations that impact Runtime Fabric core software.
+
+Consider the following points:
+
+* Audit logs are stored in a Kubernetes secret in the `rtfctl-audit` namespace.
+* Audit logs are cluster scoped and not attached to Runtime Fabric core software lifecycle.
+* Command executions continue to function if the execution fails to audit the operation.
```

Comment:
Command executions remain operational even if the auditing process fails to audit the operation